app/controllers/api/services_controller.rb

```diff
@@ -37,9 +37,9 @@ def settings
   def create
     @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
```

Comment:
Does `current_user.accessible_services.new` add anything to `@account.services.new`?
If not then it can be done in the service object

Reply:
Same as https://github.com/3scale/porta/pull/1168#discussion_r322262326
This is not a simple topic so leave it for my PR plz

Reply:
okidoki :)